db/migrate/20210809071351_deduplicate_accounts.rb

```diff
@@ -0,0 +1,14 @@
+class DeduplicateAccounts < ActiveRecord::Migration[5.2]
+  def up
+    Account.transaction do
+      DuplicateAccountResolver.run!
+      remove_index :accounts, :account_number
+      add_index :accounts, :account_number, unique: true
+    end
```

Comment:
No need for an explicit transaction. From https://edgeguides.rubyonrails.org/active_record_migrations.html#migration-overview:

> On databases that support transactions with statements that change the schema, migrations are wrapped in a transaction. If the database does not support this then when a migration fails the parts of it that succeeded will not be rolled back. You will have to rollback the changes that were made by hand.

Reply:
Oh, :facepalm: okay, it will :sweat_smile:

Reply:
@skateman maybe you forgot to push this update?